src/renderer/screens/exchange/Swap2/Form/FormSelectors/ToRow.js

```diff
@@ -126,11 +126,14 @@ export default function ToRow({
             fontWeight={600}
             color="palette.text.shade40"
             containerProps={amountInputContainerProps}
-            unit={unit}
+            /* FIXME: Currently input only accepts 16 decimals, values with more
+              decimals are cutted, that's why I apply this fix */
+            unit={unit && unit?.magnitude > 16 ? { ...unit, magnitude: 16 } : unit}
```

Comment:
We can't do this 😱 
What do you mean they are cut?
<img width="345" alt="image" src="https://user-images.githubusercontent.com/4631227/133797664-879cccde-bbb5-49e4-acea-18670bf809a7.png">